grafana/grafana
# Public Dashboards: Pubdash panels get data from pubdash api

**What this does**
Panels on a public dashboard get their data from the pubdash endpoint. This endpoint grabs the queries from the dashboard in the backend, runs them, and returns the panel data.

**How to test**
The testing process hasn't changed from previous PRs. Just make a public dashboard and see that the outgoing panel queries get sent to a new endpoint.
